lighthouse/README.md

```diff
@@ -0,0 +1,81 @@
+## Overview
+
+Get metrics from [Google Chrome Lighthouse][1] in real time to:
+* Visualize and monitor Lighthouse stats
+* Track and audit scores for your websites accessibility, best practices, performance, PWA, and SEO audit scores
+
+## Setup
+
+The Lighthouse check is not included in the [Datadog Agent][2] package, so you will need to install it yourself.
+
+### Installation
+
+1. [Download the Datadog Agent][2].
+2. Download the [`lighthouse.py` file][3] for Lighthouse.
+3. Place it in the Agent's `checks.d` directory.
+
+To install the Lighthouse Check on your host:
+
+### Configuration
+
+1. Create a `lighthouse.d/` folder in the `conf.d/` folder at the root of your Agent's directory. 
+2. Create a `conf.yaml` file in the `lighthouse.d/` folder previously created.
+3. Reference the example lighthouse [`conf.yaml.example` file][4] in this repository and copy it's contents into the `conf.yaml` file just created.
+4. Edit the `conf.yaml` file for your use case.  **Note**: It's unclear how resource intensive this can be, a typical lighthouse report takes 5-10seconds to generate so it may be necessary to have a higher than usual `minimum_collection_interval`.
```

Comment:
```suggestion
4. Edit the `conf.yaml` file for your use case.  **Note**: It's unclear how resource intensive this can be, a typical lighthouse report takes 5-10 seconds to generate so it may be necessary to have a higher than usual `minimum_collection_interval`.
```